core/src/main/resources/lib/layout/breadcrumbs.js

```diff
@@ -24,7 +24,16 @@ var breadcrumbs = (function() {
     // logger = function() { console.log.apply(console,arguments) };  // uncomment this line to enable logging
 
     function makeMenuHtml(icon,displayName) {
-        return (icon!=null ? "<img src='"+icon+"' width=24 height=24 style='margin: 2px;' alt=''> " : "")+displayName;
+        var displaynameSpan = '<span>' + displayName + '</span>';
+        if (icon === null) return "<span style='margin: 2px 4px 2px 2px;' />" + displaynameSpan;
+
+        // TODO: move this to the API response in a clean way
+        var isSvgSprite = icon.toLowerCase().indexOf('svg#') !== -1;
+        return isSvgSprite
+            ? "<svg class='svg-icon' width='24' height='24' style='margin: 2px 4px 2px 2px;' aria-label='' focusable='false'>" +
```

Comment:
can we use a template literal here for the string formatting? I can see it's not supported by IE :( https://caniuse.com/?search=template%20literal but is it transpiled?

Reply:
We cannot because that file is not going through webpack, it's an adjunct